### ISSUE TITLE
where are your footballer models stored?

### ISSUE THREAD

**Issue by OdysseyCreations** [BazkieBumpercar/GameplayFootball#6]
I have searched all the files but haven't found any 3D format files like .fbx or .max or .maya nothing then which models the players have?

**Comment by BazkieBumpercar** [comment]
Yes, it uses the .ASE files as models, they are exports from 3D Studio Max (not sure which version I used).
The .anim files are my own format, and are created with my own anim editor, which I think is included within the blunted2 github archive (not sure though).

I can send you the .max file for the player model, but keep in mind that I encoded bones/skinning variables into the vertex colors in a pretty obscure way :P If you want to use my model in another engine, you can just ignore this information and redo the skinning using the software from that engine.

Cheers!

[fullbody.max.zip](https://github.com/BazkieBumpercar/GameplayFootball/files/3223372/fullbody.max.zip)

ps. I don't have 3DS Max installed atm, but I'm 99% sure this is the right model file. It comes with no warranties etc, and under the same license (basically, public domain) as the rest of the project.

**Comment by OdysseyCreations** [comment]
Sorry to but will you pls tell me why did you discontinued such a piece of art?

**Comment by m1ga** [comment]
> Sorry to but will you pls tell me why did you discontinued such a piece of art?

it's "alive" again in a google-research project :) https://github.com/google-research/football

**Comment by BazkieBumpercar** [comment]
Yes I am aware, I have been contacted about this by Google. It's really cool my project found a use after all!
Cheers, Bastiaan

**Comment by roberth11** [comment]
could you pass me the compiled animator since the one that goes up in blunted throws errors

**Comment by roberth11** [comment]
> Sí, utiliza los archivos .ASE como modelos, son exportaciones de 3D Studio Max (no estoy seguro de qué versión utilicé).
> Los archivos .anim son de mi propio formato y están creados con mi propio editor de animación, que creo que está incluido en el archivo de github blunted2 (aunque no estoy seguro).
> 
> Puedo enviarle el archivo .max para el modelo de reproductor, pero tenga en cuenta que codifiqué las variables huesos / piel en los colores de vértice de una manera bastante oscura: P Si desea usar mi modelo en otro motor, puede ignorar esta información y rehacer el skinning usando el software de ese motor.
> 
> ¡Salud!
> 
> [fullbody.max.zip](https://github.com/BazkieBumpercar/GameplayFootball/files/3223372/fullbody.max.zip)
> 
> PD. No tengo instalado el cajero automático 3DS Max, pero estoy 99% seguro de que este es el archivo de modelo correcto. Viene sin garantías, etc., y bajo la misma licencia (básicamente, dominio público) que el resto del proyecto.

podrias pasar todos los archivos que utilizaste tanto para juggador como para estadio y balon gracias

**Comment by hasanleiva** [comment]
How we can open that .anim files?
Can we compile anim tool which include blunted2?

**Comment by roberth11** [comment]
> ¿Cómo podemos abrir esos archivos .anim?
> ¿Podemos compilar una herramienta de animación que incluya blunted2?

you already managed to compile the game
